specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json

```diff
@@ -894,7 +894,7 @@
                 "type": "string"
               },
               "score": {
-                "type": "integer"
+                "type": "float"
```

Comment:
This is not the right way to document a float
https://swagger.io/docs/specification/data-models/data-types/